dispatcher/DAQController.py

```diff
@@ -361,6 +361,8 @@ def CheckTimeouts(self, detector, command = None):
                                          "can't solve"),
                                         'ERROR',
                                         "STOP_TIMEOUT")
+                    # also invoke the nuclear option
```

Comment:
What would be the context and the desired result here?

Reply:
The context is that the system isn't responding to dispatcher commands if we get here, and the desired result is that the problem gets fixed.